nvflare/private/fed/server/server_engine.py

```diff
@@ -652,7 +652,10 @@ def persist_components(self, fl_ctx: FLContext, completed: bool):
 
             self.server.snapshot_location = self.snapshot_persistor.save(snapshot=snapshot)
         finally:
-            self.logger.info(f"persist the snapshot to: {self.server.snapshot_location}")
+            if not completed:
+                self.logger.info(f"persist the snapshot to: {self.server.snapshot_location}")
+            else:
+                self.logger.info(f"The snapshot: {self.server.snapshot_location} has been removed.")
```

Comment:
`self.snapshot_persistor.save(snapshot=snapshot)` this is not guaranteed to finish successfully in the try block.

So the print statement in this finally block is not stating the fact.

I think we need to move the print inside the try block.

Reply:
done.